src/AssetManager/Service/AssetFilterManagerServiceFactory.php

```diff
@@ -2,30 +2,39 @@
 
 namespace AssetManager\Service;
 
+use Interop\Container\ContainerInterface;
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
 class AssetFilterManagerServiceFactory implements FactoryInterface
 {
     /**
-     * {@inheritDoc}
-     *
-     * @return AssetFilterManager
+     * @inheritDoc
      */
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
         $filters = array();
-        $config  = $serviceLocator->get('Config');
+        $config  = $container->get('config');
 
         if (!empty($config['asset_manager']['filters'])) {
             $filters = $config['asset_manager']['filters'];
         }
 
         $assetFilterManager = new AssetFilterManager($filters);
 
-        $assetFilterManager->setServiceLocator($serviceLocator);
-        $assetFilterManager->setMimeResolver($serviceLocator->get('AssetManager\Service\MimeResolver'));
+        $assetFilterManager->setServiceLocator($container);
+        $assetFilterManager->setMimeResolver($container->get(MimeResolver::class));
 
         return $assetFilterManager;
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @return AssetFilterManager
+     */
+    public function createService(ServiceLocatorInterface $serviceLocator)
```

Comment:
If we're switching to container-interop Shouldn't this type hint against that instead?